include/core/parameters_lagrangian.h

```diff
@@ -203,13 +211,7 @@ namespace Parameters
       double neighborhood_threshold;
 
       // Particle-particle contact force model
-      enum class ParticleParticleContactForceModel
-      {
-        linear,
-        hertz_mindlin_limit_force,
-        hertz_mindlin_limit_overlap,
-        hertz
-      } particle_particle_contact_force_method;
+      ParticleParticleContactForceModel particle_particle_contact_force_method;
```

Comment:
I just moved it outside of the very nested parameter namespace, otherwise accessing it was becoming really a pain. Ideally I would maybe want to move it outside of the parameter namespace and move it to the .h of the contact model, but I would rather not do that now.